src/dropdown-arrow-down.js

```diff
@@ -0,0 +1,13 @@
+import { createElement, Component } from 'preact' /** @jsx createElement */
+
+export default class DropdownArrowDown extends Component {
+  render () {
+    return (
+      <svg version='1.1' xmlns='http://www.w3.org/2000/svg' class='dropdown-arrow-down'>
```

Comment:
For the className itself, we should use a BEMmy one to keep with the other ones:

```js
const dropdownArrowClassName `${cssNamespace}__dropdown-arrow`
```

Which is `.autocomplete__dropdown-arrow` by default.